expertise/service/__init__.py

```diff
@@ -73,3 +77,11 @@ def create_celery(app):
     )
 
     return celery
+
+
+def create_redis(app):
+    """
+    Initializes a redis connection pool
+    """
+    pool = redis.ConnectionPool(host='localhost', port=6379, db=5)
```

Comment:
I think we should store these in the Flask config so we know where all the hardcoded parameters are, here's how I did them in my PR: https://github.com/openreview/openreview-expertise/blob/cb52038ca6adff2430b0aec41945985457822e42/expertise/service/__init__.py#L83-L87

Will we need 2 connection pools since you chose to use db 5 and mine is currently on db 10?